README.md

```diff
@@ -89,6 +88,27 @@ See [Safely open apps on your Mac](https://support.apple.com/en-us/HT202491) for
 Depending on your security settings, you may also get a panel asking if you wish to allow it to accept
 network connections. You can click "Deny" as we do not need network access currently nor we do request any.
 
+### Package Manager ###
+
+`Logisim-evolution` is available from a bunch of package managers.  
+Note that these (except of Snap) exare not maintained by the core developers.  
```

Comment:
"exare" -> "are"

Reply:
(except of Snap) should be (except for Snap)